compiler_opt/rl/imitation_learning/generate_bc_trajectories_lib.py

```diff
@@ -918,6 +927,10 @@ def gen_trajectories(
     worker_manager_class: A pool of workers hosted on the local machines, each
       in its own process.
   """
+  if (None in (_BASE_PATH.value, FLAGS.keep_temps) and
+      not all(el is None for el in (_BASE_PATH.value, FLAGS.keep_temps))):
+    raise ValueError(('Both flags keep_temps={FLAGS.keep_temps} and'
```

Comment:
I think the if statement could probably be simpler if written without all the Python syntactic sugar?

```python
if (_BASE_PATH.value is None or FLAGS.keep_temps is None) and (_BASE_PATH.value is not None and FLAGS.keep_temps is not None):
  pass
```

They're both understandable, but I think this is a quite a bit more readable.